packages/core/utils/src/modules-sdk/migration-scripts/migration-up.ts

```diff
@@ -52,6 +53,7 @@ export function buildMigrationScript({ moduleName, pathToMigrations }) {
       }
     } catch (error) {
       logger.error(`Failed with error ${error.message}`, error)
+      throw new MedusaError(MedusaError.Types.DB_ERROR, error.message)
```

Comment:
Let's just make sure we don't double-log the error somewhere higher up the stack, otherwise LGTM

Reply:
it happens that the command log the error too, but these scripts are also called in other places. Not sure this one or other ones should know about which one is logging. If you feel I should remove it still let me know